templates/components/search/displaypreference_config.html.twig

```diff
@@ -0,0 +1,227 @@
+{#
+ # ---------------------------------------------------------------------
+ #
+ # GLPI - Gestionnaire Libre de Parc Informatique
+ #
+ # http://glpi-project.org
+ #
+ # @copyright 2015-2023 Teclib' and contributors.
+ # @copyright 2003-2014 by the INDEPNET Development Team.
+ # @licence   https://www.gnu.org/licenses/gpl-3.0.html
+ #
+ # ---------------------------------------------------------------------
+ #
+ # LICENSE
+ #
+ # This file is part of GLPI.
+ #
+ # This program is free software: you can redistribute it and/or modify
+ # it under the terms of the GNU General Public License as published by
+ # the Free Software Foundation, either version 3 of the License, or
+ # (at your option) any later version.
+ #
+ # This program is distributed in the hope that it will be useful,
+ # but WITHOUT ANY WARRANTY; without even the implied warranty of
+ # MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ # GNU General Public License for more details.
+ #
+ # You should have received a copy of the GNU General Public License
+ # along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ #
+ # ---------------------------------------------------------------------
+ #}
+
+{% import 'components/form/fields_macros.html.twig' as fields %}
+
+<div id="tabsbody" class="m-n2 display_preference_config">
+   <input type="hidden" name="itemtype" value="{{ itemtype }}">
+   <input type="hidden" name="users_id" value="{{ users_id }}">
+   {% if not available_itemtype %}
+      <div class="alert alert-danger">
+         {{ __('These preferences cannot be edited at this time. The item type no longer exists or is provided by a plugin that is not enabled.') }}
+      </div>
+   {% else %}
+      {% if is_global %}
+         <div class="alert alert-info">
+            {{ __('These preferences are used by everyone that does not have a personal view configured.') }}
+         </div>
+      {% endif %}
+      {% if not is_global and not has_personal %}
+         <div class="alert alert-info">
+            {{ __('No personal criteria. Create personal parameters?') }}
+         </div>
+         <button type="button" class="btn btn-primary" name="activate" value="1">{{ __('Create') }}</button>
+      {% else %}
+         <table class="table table-striped card-table m-n2">
```

Comment:
are a table mandatory here ? If yes, at least remove `.m-n2` and move header stuff outside the table
if no, could we move to a ul > li design (use flex to make delete button at the right) ?